[IMP] rental: revise physical rental products page format (#19282)
odoo/documentation: content/applications/sales/rental/configure_products/products.rst

Reviewer: ```suggestion
Add the appropriate :ref:`attribute and its values <products/variants/attributes>` by clicking
```
We either need to remove the "the" here to make the sentence grammatically correct, or we need to add the word "link" after. This solution causes the least amount of reconfiguring of lines.

Reviewer: ```suggestion
      :alt: Sample of a rental product with the custom rental pricelist applied.
```
One too many spaces indented here.

Reviewer: ```suggestion
   pick-up and drop-off of their bikes. The company wants to track its bikes accurately at each
```
This should make it easier for translation -- I realize we sometimes use "Pickedup" or "Pickup" in the UI, but that's grammatically incorrect.

Reviewer: ```suggestion
   Transfers` in the **Rental** app and :guilabel:`Storage Locations` in the **Inventory** app.
```

Reviewer: ```suggestion
To create pickup and/or return receipts, navigate to the desired rental order, and click the
```

Reviewer: ```suggestion
```
One too many blank lines at the end of the file. We just need one.

displayed for every :guilabel:`Periodicity` field option except :guilabel:`Hours`. The
:guilabel:`Pickup` and :guilabel:`Return` times only apply to online rental orders.

All additional rental rates can be configured on the :ref:`rental/products/multiple-rental-prices`,
but they are restricted to the configured :guilabel:`Periodicity` value selected in the *Sales* tab.
In other words, a rental product can only have one :guilabel:`Periodicity` value configured at a

> time.

Reviewer: ```suggestion
All additional rental rates can be configured when setting up :ref:`multiple rental prices
<rental/products/multiple-rental-prices>`, but they are restricted to the configured
:guilabel:`Periodicity` value selected in the *Sales* tab. In other words, a rental product can only
have one :guilabel:`Periodicity` value configured at a time.
```
The ref link you created makes the sentence read awkwardly, so I added a label to it. If it could be clearer, feel free to modify, but it needs something other than the pulled-in header in the middle of that sentence.

tracking and differentiating the inventory, and providing more detailed reports. Examples of rental
variants for a *Goods* product are: sizes, brand, color, and material.

.. _rental/products/multiple-rental-prices:

Add multiple rental prices
--------------------------

.. important::

>    The **Sales** app must be installed, the *Pricelist* feature must be enabled for this tab to

Reviewer: ```suggestion
   The **Sales** app must be installed, and the *Pricelist* feature must be enabled for this tab to
```

documentation.

.. tip::
   It is recommended to create a new :guilabel:`Pricelist` first, then select the customized
   :guilabel:`Pricelist` in the *Prices* tab instead of using the :guilabel:`Default` pricelist.
   Keeping the *Default* pricelist blank ensures there is a clean pricelist for the base rental
   rate.

.. _rental/products/pricelist-method:

Using the Pricelists method
~~~~~~~~~~~~~~~~~~~~~~~~~~~

Creating a :ref:`new Pricelist <sales/products/create-edit-pricelists>` allows for better
customization when applying rental rates to specific time periods, products, or customers by using
Pricelist Rules. It is a separate form that users can apply on quotations or select on the rental
product form to add new price rules to.

Navigate to :menuselection:`Rental app --> Products --> Pricelists` and click :guilabel:`New`. A
*Create Pricelist Rules* window displays.

.. image:: products/create-pricelist-rules-window.png
   :alt: Sample of the Create Pricelist Rules window in the Odoo Rental application.

.. _rental/products/price-tab-method:

Using the Prices tab method
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Rental rates can also be configured as a new price rule to an existing pricelist using the

Reviewer: ```suggestion
Rental rates can also be configured as a new price rule for an existing pricelist using the
```
This preposition reads better.

.. _rental/products/product-tracking:

Configure product tracking
==========================

.. important::
   To configure a physical rental product for product tracking, the **Inventory** app must be
   installed, and :guilabel:`Lots & Serial Numbers` must be enabled.

   For rental products, only :guilabel:`By Unique Serial Number` tracking can be used. Selecting

>    :guilabel:`By Lots` instead results in an error message, since that setting doesn’t apply to

Reviewer: ```suggestion
   :guilabel:`By Lots` instead results in an error message, since that setting doesn't apply to
```
Watch out for the curly apostrophes. :)

   rentals.

Go to the :menuselection:`Rental app --> Products --> Products`, then click :guilabel:`New`. In the
new product window, the :guilabel:`Sales` checkbox is already selected by default. Select
:guilabel:`Goods` as the :guilabel:`Product Type`, then select the :guilabel:`Track Inventory`
checkbox. The :guilabel:`Tracking` and :guilabel:`Quantity On Hand` fields display.

Click into the :guilabel:`Tracking` field and select :guilabel:`By Unique Serial Number`. Enter the
number of products available to rent in the :guilabel:`Quantity On Hand` field.

For the :guilabel:`Category` field, select :guilabel:`Goods` from the drop-down menu or create a new
category by typing in the name and clicking :guilabel:`Create`. Configure :ref:`basic rental rate

> <rental/products/base-rental-period-price>` and any :ref:`additional rates

Reviewer: ```suggestion
category by typing in the name and clicking :guilabel:`Create`. Configure the :ref:`basic rental
rate <rental/products/base-rental-period-price>` and any :ref:`additional rates
```
Missing a word here.

=======================================

.. important::
   Refer to the :ref:`Rental Transfers note <rental/products/rental-transfers-note>` for information

>    about internal location configuration and inventory tracking..

Reviewer: ```suggestion
   about internal location configuration and inventory tracking.
```
One too many periods.
